src/Symfony/Component/Notifier/Bridge/Smsbox/.gitignore

```diff
@@ -0,0 +1,3 @@
+vendor/
+composer.lock
+phpunit.xml
```

Comment:
please add a newline to make the red pictogram disapear

Reply:
fixed